fsharp/oxpecker/Program.fs

```diff
@@ -0,0 +1,41 @@
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.DependencyInjection
+open Microsoft.Extensions.Hosting
+open Microsoft.Extensions.Logging
+open Oxpecker
+
+let emptyHandler: EndpointHandler = fun _ -> Task.CompletedTask
+
+let userIdHandler: EndpointHandler =
+    fun ctx ->
+        match ctx.TryGetRouteValue<string>("id") with
```

Comment:
I think id is not optional in this benchmark, so it can be simplified to
```fsharp
let id = ctx.Request.RouteValues["id"] :?> string
ctx.WriteText(id)
```